.github/workflows/build-and-test-snap.yml

```diff
@@ -31,19 +57,12 @@ jobs:
         env:
           MOCK_GPIO: true
           SKIP_TEARDOWN_REMOVAL: true
-          LOCAL_SERVICE_SNAP: ../${{ steps.snapcraft.outputs.snap }}
-        run: go test -failfast -p 1 -timeout 20m -v
+          LOCAL_SERVICE_SNAP: ../matter-pi-gpio-commander_2.0.0_arm64.snap
+        run: ls -la && go test -failfast -p 1 -timeout 20m -v
```

Comment:
The snap version should not be hardcoded here.
Please remove debugging code (ls -la).